test/test_util.py

```diff
@@ -48,6 +48,7 @@
 # numbers used for timeouts
```

Comment:
Ian is saying that there are spurious changes here *beyond* the change to tuple, which he's just OK'd.